contrib/nydusify/cmd/nydusify_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 Alibaba Cloud. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Maybe use `github.com/stretchr/testify/require` to stop the test once the error happens.

Reply:
Thank you, your suggestion is very useful.